python-modules/cis_aws/cis_aws/connect.py

```diff
@@ -156,8 +153,10 @@ def input_stream_client(self):
     def _check_sessions_exist(self):
         if self._discover_cis_environment() == 'local':
             logger.info('CIS Local environment detected skipping cloud based validations.')
+            return
```

Comment:
why not `elif`?

Reply:
where would the `elif` go?

Reply:
ah I see... never-mind.  I'll change this.  But would say this is a style choice.